source/slang/slang-emit-c-like.h

```diff
@@ -405,15 +407,15 @@ class CLikeSourceEmitter: public SourceEmitterBase
     void emitStruct(IRStructType* structType);
     // This is used independently of `emitStruct` by some GLSL parameter group
     // output functionality
-    void emitStructDeclarationsBlock(IRStructType* structType);
+    void emitStructDeclarationsBlock(IRStructType* structType, bool allowOffsetLayout);
```

Comment:
Would be nice to have an enum for this, otherwise the calling code has a contextless true/false

Reply:
likewise above